llvm/utils/lit/tests/shtest-env.py

```diff
@@ -9,35 +9,35 @@
 
 # CHECK: -- Testing: 16 tests{{.*}}
 
-# CHECK: FAIL: shtest-env :: env-args-last-is-assign.txt ({{[^)]*}})
+# CHECK: PASS: shtest-env :: env-args-last-is-assign.txt ({{[^)]*}})
 # CHECK: env FOO=1
 # CHECK: # executed command: env FOO=1
-# CHECK: # | Error: 'env' requires a subcommand
-# CHECK: # error: command failed with exit status: {{.*}}
+# CHECK-NOT: {{^[^#]}}
+# CHECK: --
```

Comment:
Is this on the next line, if so we can simplify these passing tests using `CHECK-NEXT: --` instead of the CHECK-NOT: followed by CHECK.

Reply:
Yes, if the expected lines are sequential and should appear in the exact order without any unexpected lines in between, we can simplify these passing tests using CHECK-NEXT: instead of CHECK-NOT: followed by CHECK. I will change it right now.

Reply:
What exactly is the output that you get in these cases?

Reply:
(As an aside, I see this pattern is followed below, so I'm not suggesting you change it now).